client/scripts/views/modals/create_community_modal.ts

```diff
@@ -134,33 +139,212 @@ const CreateCommunityModal: m.Component<IAttrs, IState> = {
+  oninit: (vnode) => {
+    vnode.state.name = '';
+    vnode.state.nodeUrl = '';
+    vnode.state.symbol = '';
+    vnode.state.website = '';
+    vnode.state.discord = '';
+    vnode.state.element = '';
+    vnode.state.telegram = '';
+    vnode.state.github = '';
+    vnode.state.description = '';
+    vnode.state.substrate_spec = '';
+  },
+  view: (vnode) => {
+    return m('.compact-modal-body-max', [
+      m('.CommunityMetadataManagementTable', [m(Table, {
+        bordered: false,
+        interactive: false,
+        striped: false,
+        class: 'metadata-management-table',
+      }, [
+        m(InputPropertyRow, {
+          title: 'Name',
+          defaultValue: vnode.state.name,
+          onChangeHandler: (v) => { vnode.state.name = v; },
+        }),
+        m(InputPropertyRow, {
+          title: 'Node URL',
+          defaultValue: vnode.state.nodeUrl,
+          placeholder: 'wss://',
+          onChangeHandler: (v) => { vnode.state.nodeUrl = v; },
+        }),
+        m(InputPropertyRow, {
+          title: 'Symbol',
+          defaultValue: vnode.state.symbol,
+          placeholder: 'XYZ',
+          onChangeHandler: (v) => { vnode.state.symbol = v; },
+        }),
+        m(InputPropertyRow, {
+          title: 'Description',
+          defaultValue: vnode.state.description,
+          onChangeHandler: (v) => { vnode.state.description = v; },
+          textarea: true,
+        }),
+        m(InputPropertyRow, {
+          title: 'Spec (JSON)',
+          defaultValue: vnode.state.substrate_spec,
+          textarea: true,
+          placeholder: `{"types": {"Address": "MultiAddress", "ChainId": "u8", "Reveals": "Vec<(AccountId, Vec<VoteOutcome>)>", "Balance2": "u128", "VoteData": {"stage": "VoteStage", "initiator": "AccountId", "vote_type": "VoteType", "tally_type": "TallyType", "is_commit_reveal": "bool"}, "VoteType": {"_enum": ["Binary", "MultiOption", "RankedChoice"]}, "TallyType": {"_enum": ["OnePerson", "OneCoin"]}, "VoteStage": {"_enum": ["PreVoting", "Commit", "Voting", "Completed"]}, "ResourceId": "[u8; 32]", "VoteRecord": {"id": "u64", "data": "VoteData", "reveals": "Reveals", "outcomes": "Vec<VoteOutcome>", "commitments": "Commitments"}, "AccountInfo": "AccountInfoWithRefCount", "Commitments": "Vec<(AccountId, VoteOutcome)>", "VoteOutcome": "[u8; 32]", "VotingTally": "Option<Vec<(VoteOutcome, u128)>>", "DepositNonce": "u64", "LookupSource": "MultiAddress", "ProposalTitle": "Bytes", "ProposalVotes": {"staus": "ProposalStatus", "expiry": "BlockNumber", "votes_for": "Vec<AccountId>", "votes_against": "Vec<AccountId>"}, "ProposalRecord": {"index": "u32", "stage": "VoteStage", "title": "Text", "author": "AccountId", "vote_id": "u64", "contents": "Text", "transition_time": "u32"}, "ProposalStatus": {"_enum": ["Initiated", "Approved", "Rejected"]}, "ProposalContents": "Bytes"}}`,
+          onChangeHandler: (v) => { vnode.state.substrate_spec = v; },
```

Comment:
this validation needs to be more like the one in the `spec_settings.ts` file. And we should also add the "Test Button" to make sure that the spec and everything works (connects to chain via api). This should mostly be copy/pasta work.